spring/src/main/java/org/axonframework/spring/config/AbstractAnnotationHandlerBeanPostProcessor.java

```diff
@@ -70,7 +70,7 @@ public Object postProcessBeforeInitialization(Object bean, String beanName) thro
      */
     @Override
     public Object postProcessAfterInitialization(final Object bean, final String beanName) throws BeansException {
-        if (beanName != null && beanFactory.containsBean(beanName) && !beanFactory.isSingleton(beanName)) {
+        if (beanName != null && !bean.equals(null) && beanFactory.containsBean(beanName) && !beanFactory.isSingleton(beanName)) {
```

Comment:
!bean.equals(null) -> lol ?

Reply:
https://github.com/spring-projects/spring-framework/blob/v5.2.4.RELEASE/spring-beans/src/main/java/org/springframework/beans/factory/support/NullBean.java#L44

(bean == null) != true

Reply:
I fully agree with @Blackdread that's this change incites a "lol" or "wtf".

Please move this check to a private method with Javadoc describing why this is necessary in the first place.

To be honest it feels **extremely** off that such a repo as you're including returns `null` beans on the subject. However it's a fact of live apparently, so we'll have to deal with it.
 
Thus again, describing the necessity of this "weirdness" and including a link to to both references you've shared too.

Reply:
Understood. Will do so, thanks

Reply:
Thanks you very much @cmyker :bowing_man: